package.json

```diff
@@ -105,7 +106,8 @@
     "source-map-explorer": "^1.6.0",
     "ts-jest": "^23.10.3",
     "ts-loader": "^5.2.1",
-    "typescript": "^3.1.1"
+    "typescript": "^3.1.1",
+    "typings-for-css-modules-loader": "^1.7.0"
```

Comment:
This is actually pretty cool, I just wish it didn't have to write to disk to work :blobcatverysad:

Would be nice if it'd work as a typescript plugin.